packages/cli/CHANGELOG.md

```diff
@@ -1,24 +1,31 @@
-# Change Log
-
-All notable changes to the Client code for "zowe-native-proto-cli" are documented in this file.
-
-Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
-
-## `0.1.1`
-
-- Fixed issue where a jobs list request returns unexpected results whenever a search query does not match any jobs. [#217](https://github.com/zowe/zowe-native-proto/pull/217)
-
-## `0.1.0`
-
-- Added `zssh restore dataset` command. [#38](https://github.com/zowe/zowe-native-proto/pull/38)
-- Added `zssh view uss-file` command. [#38](https://github.com/zowe/zowe-native-proto/pull/38)
-- Added `zssh list spool-files` command. [#38](https://github.com/zowe/zowe-native-proto/pull/38)
-- Added support for cancelling jobs. [#138](https://github.com/zowe/zowe-native-proto/pull/138)
-- Added support for holding and releasing jobs. [#182](https://github.com/zowe/zowe-native-proto/pull/182)
-- Updated error handling for listing data sets. [#185](https://github.com/zowe/zowe-native-proto/pull/185)
-- Added `zssh submit uss-file|local-file|data-set` commands. [#184](https://github.com/zowe/zowe-native-proto/pull/184)
-- Added support for conflict detection through use of e-tags. When a data set or USS file is viewed, the e-tag is displayed to the user and can be passed for future write requests to prevent overwriting new changes on the target system. [#144](https://github.com/zowe/zowe-native-proto/issues/144)
-
-## [Unreleased]
-
-- Initial release
+# Change Log
+
+All notable changes to the Client code for "zowe-native-proto-cli" are documented in this file.
+
+Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
+
+## Recent Changes
+
+- Added support for listing the long format for USS files with the `long` request option. When provided, the CLI plug-in prints additional file attributes, including permissions, file tag, file size, number of links, owner, and group. [#346](https://github.com/zowe/zowe-native-proto/issues/346)
+- Added support for listing all USS files with the `all` request option. When provided, the CLI plug-in includes hidden files in the list response. [#421](https://github.com/zowe/zowe-native-proto/pull/421)
+- Added support for the `encoding` option to the `zssh upload uss` and the `zssh upload ds` commands. When the `file` option is provided, the command uses the option value as the target encoding for the uploaded content. [#427](https://github.com/zowe/zowe-native-proto/issues/427)
+- Added support for the `file` option to the `zssh download uss` and `zssh download ds` commands. When the `file` option is provided, the command uses the option value as the destination file path for the downloaded content. [#428](https://github.com/zowe/zowe-native-proto/issues/428)

```

Comment:
Should we warn people if the destination file path already exists?
it seems that we just override it 😅 

---

The PR title talks about auto-detect encoding, however, even for tagged files, I still need to specify the --encoding flag 🤔

Reply:
If we want similar behavior to Zowe CLI, we should probably have an `--override` option or `--force` option that forcefully overwrites the file. If the option is not provided and the file exists, we can display an error message and recommend this option. I can implement this but this would expand the work a bit more outside the scope of this story/PR (I know I've already added some stuff in here that wasn't a part of the story, just want to avoid further scope creep 😅)

Reply:
>> Should we warn people if the destination file path already exists?

> If we want similar behavior to Zowe CLI, we should probably have an --overwrite option or --force option that forcefully overwrites the file.

no worries, we can create a separate issue for this 🙏